core/src/main/scala/io/projectglow/Glow.scala

```diff
@@ -51,15 +56,20 @@ class GlowBase {
    *
    * @return The transformed DataFrame
    */
-  def transform(operationName: String, df: DataFrame, options: Map[String, String]): DataFrame = {
+  def transform(operationName: String, df: DataFrame, options: Map[String, Any]): DataFrame = {
+    //
+    val stringValuedMap = options.mapValues {
+      case s: String => s
```

Comment:
Do we need this?

Reply:
Yes, otherwise strings are formatted like `"string"`